scripts/releaseV2/phase2-bump-all-packages.mjs

```diff
@@ -46,19 +46,7 @@ const rootFolder = resolve(dirname(fileURLToPath(import.meta.url)), '..', '..');
     await writeChangelog(PATH, changelog);
   }
 
-  // TODO: Revert spawnSync to npmPublish.
   npmPublish();
-  const publish = spawnSync(appendCmdIfWindows`npm`, ['publish'], {
-    cwd: undefined,
-  });
-  publish.stdout
-    .toString()
-    .split('\n')
-    .forEach((line) => console.log(line));
-  publish.stderr
-    .toString()
-    .split('\n')
-    .forEach((line) => console.error(line));
```

Comment:
We were publishing twice. Luckily, the second one ain't throwing the main thread, so the release was working, despite it always failing.